pytd/client.py

```diff
@@ -35,9 +35,15 @@ class Client(object):
     header : string or boolean, default: True
         Prepend comment strings, in the form "-- comment", as a header of queries.
         Set False to disable header.
+
+    writer : pytd.writer.Writer or child class of Writer class, optional
+        A Writer to choose writing method to Treasure Data. If not given, default Writer
+        will be created with executing :func:`~pytd.Client.load_table_from_dataframe`
+        at the first time.
     """
 
-    def __init__(self, apikey=None, endpoint=None, database='sample_datasets', engine='presto', header=True, **kwargs):
+    def __init__(
+            self, apikey=None, endpoint=None, database='sample_datasets', engine='presto', header=True, writer=None, **kwargs):
```

Comment:
I'd love to avoid formatting discussion, so it'd be nice if we could have another PR with adding black.

Reply:
possible bikeshedding - it might be worth considering to setup [pre-commit](https://pre-commit.com/) and its [yapf plugin](https://github.com/pre-commit/mirrors-yapf) so that formatting style will be some automated task, not something derived from individual's preference :smile:

Reply:
Noted 📝